tools/devops/automation/templates/main-stage.yml

```diff
@@ -32,10 +32,6 @@ parameters:
   type: boolean
   default: true
 
-- name: runTranslations
-  type: boolean
-  default: true
```

Comment:
leave the parameter, and remove it after you create a pr in https://github.com/xamarin/sdk-insertions updating the usage, else this will affect the work in that other repo.

So:

1. Remove  the step (in this PR) but leave the param.
2. Update the usage in https://github.com/xamarin/sdk-insertions both in main
3. Remove the parameter from here (new PR).

Reply:
Created this issue here to help with this: https://github.com/xamarin/xamarin-macios/issues/17870